### PR TITLE
Adding common_data_pool role

### PR DESCRIPTION
**Pull request by Alexey-ebi**
It is PR to add common data pool. I've chosen to expose port 26081 and use data_portal path. This code will add only DRF to server.

### CODE REVIEW

File: roles/image/templates/image.conf.j2 [hunk 1]
```diff
@@ -93,6 +93,31 @@ server {
     proxy_redirect http://$host:$my_port/ $scheme://$http_host/;
   }
 
+  location /data_portal/ {
+    # check if site is under mantainance
+    # Se trovo questo file, ritorno un errore 503 a prescindere
+    if (-f /usr/share/nginx/html/image_maintenance.html) {
```
Comment (bunop) [inline roles/image/templates/image.conf.j2]: Could you give this file a different name? You will turn off all location with the same file name.

**Review by bunop — COMMENTED** [review]
Thank you Alex! Check my comment below

File: roles/image/templates/image.conf.j2 (hunk 1 above)
Comment (Alexey-ebi) [reply roles/image/templates/image.conf.j2]: I've decided to completely remove this check as with angular app deployment of new version will always be instantaneous.

File: roles/angular_cli/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+# tasks file for angular_cli
+- name: Install required packages
+  apt:
+    name: [ 'npm' ]
```
Comment (bunop) [inline roles/angular_cli/tasks/main.yml:5]: In this case, installing the default npm using the default package manager, is sufficent, or you don't need to build your applications against a specific version npm?

**Pull request merged**